starter-code/pizza.js

```diff
@@ -1 +1,222 @@
-// Write your Pizza Builder JavaScript in this file.
+// // Write your Pizza Builder JavaScript in this file.
```

Comment:
Oh yes, I remember your different approach.  It works though, and is arguably more efficient/shorter than how many other people are doing it.